src/main/steps/applicant/upload-document/upload-your-documents/content.test.ts

```diff
@@ -7,19 +7,69 @@ import { generateContent } from './content';
 const en = {
   section: 'Provide the document',
   title: 'Provide the documents',
+  declaration: 'I believe that the facts stated in these documents are true',
+  consent: 'This confirms that the information you are submitting is true and accurate, to the best of your knowledge.',
   continue: 'Continue',
   add: 'Submit',
   uploadFiles: 'Your documents',
   remove: 'Remove',
+  textAreaDocUploadText1: 'You can use this box to:',
+  textAreaDocUploadText2: 'write a statement if you do not want to upload a document',
+  textAreaDocUploadText3: 'give the court more information about the documents you are uploading',
+  uplodFileText1:
+    'If you are uploading documents from a computer, name the files clearly. For example, letter-from-school.doc.',
+  uplodFileText2: 'Files must end with JPG, BMP, PNG,TIF, PDF, DOC or DOCX.',
+  uplodFileText3: 'How to take a picture of a document on your phone and upload it',
+  uplodFileText4: 'Place your document on a flat service in a well-lit room. Use a flash if you need to.',
+  uplodFileText5: 'Take a picture of the whole document. You should be able to see its edges.',
+  uplodFileText6: 'Check you can read all the writing, including the handwriting.',
+  uplodFileText7: 'Email or send the photo or scan to the device you are using now.',
+  uplodFileText8: 'Upload it here.',
+  uploadFileHeading: 'Upload a file',
+  uploadFile: 'Upload file',
+  statementOfTruth: 'Statement of truth',
+  warning: 'Warning',
+  warningText:
+    'Proceedings for contempt of court may be brought against anyone who makes, or causes to be made, a false statement verified by a statement of truth without an honest belief in its truth.',
+  errors: {
+    declarationCheck: {
+      required: 'Please confirm the declaration',
+    },
+  },
 };
 
 const cy: typeof en = {
```

Comment:
Language tag identifier is missing.

Reply:
This line has been removed as language content is being imported